.github/workflows/build.yml

```diff
@@ -23,9 +30,23 @@ jobs:
       - name: Build CLI
         working-directory: ./packages/cli
         run: tsc -b
+      - name: Set the template version
+        run: |
+          echo "UI_TEMPLATE_VERSION=$(node scripts/get-next-test-version.js)" >> $GITHUB_ENV
+      - name: Release test versions
+        run: |
+          echo "E2E tests should target template version ${{ env.UI_TEMPLATE_VERSION }}"
+          git config --global user.email action@github.com
+          git config --global user.name GitHub Action
+        # node_modules/.bin/lerna version ${{ env.UI_TEMPLATE_VERSION }} --no-changelog --no-push -y
```

Comment:
@louis-bompart 
I don't think this line will work because it will only bump packages that have changed since the last release.
The goal is to bump all the templates to the same 0.0.x version, so we can pass that version to the E2E tests.

Should we instead just use `npm bump` instead of lerna?

Reply:
or maybe `lerna version --force-publish=<package-1>,<package-2>,...`?